src/index.html

```diff
@@ -6,8 +6,53 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin ="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__picture"
+        src="./images/imac.jpeg"
+        alt="Apple A1419 iMac 27&quot Retina 5K Monoblock"
+      >
+
+      <div class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="card__rating">
+        <div class="stars">
+          <div class="star star--1"></div>
+
+          <div class="star star--2"></div>
+
+          <div class="star star--3"></div>
+
+          <div class="star star--4"></div>
+
+          <div class="star star--5"></div>
+        </div>
+        <div class="card__reviews">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__block-1">Price:</div>
```

Comment:
```suggestion
        <div class="card__price-label">Price:</div>
```
`block-1` and `block-2` says nothing about these elements
let's change names, to make them more descriptive, let's say `card__price-label` and `card__price-value`, etc.